src/Template/Element/advanced_search_form.ctp

```diff
@@ -347,7 +362,7 @@ echo $this->Form->create('AdvancedSearch', [
         </div>
     </div>
 
-    <div class="buttons" layout="<?= $layout ?>">
+    <div class="buttons" layout="<?= $layout ?>" style="align-items:center">
```

Comment:
This seems to be causing the search and more search options buttons to display incorrectly on the search results screen.